Makefile

```diff
@@ -9,9 +9,6 @@ ifeq (${uname},Darwin)
 	OS=darwin
 endif
 
```

Comment:
By declaring export command here, I think that `GO111MODULE=on` becomes unnecessary for each task.
```suggestion
# Enable go modules
export GO111MODULE=on
```